app/scripts/views/museum/museum-controller.js

```diff
@@ -190,6 +193,16 @@
                 searchPolygon = null;
             }
         }
+
+        function requestWithSpinner(dfd) {
```

Comment:
Sure, yeah thats a bit better. I didn't really have a good name for this one either.